exporters/jaeger-thrift/src/main/java/io/opentelemetry/exporter/jaeger/thrift/JaegerThriftSpanExporter.java

```diff
@@ -110,6 +111,12 @@ public CompletableResultCode export(Collection<SpanData> spans) {
 
   private Process createProcess(Resource resource) {
     Process result = new Process(this.process);
+
+    String serviceName = resource.getAttributes().get(ResourceAttributes.SERVICE_NAME);
+    if (serviceName != null && !serviceName.isEmpty()) {
+      result.setServiceName(serviceName);
```

Comment:
Didn't we still need to fallback to a constant in exporters (though not make that configurable)?

Reply:
that will be a follow-on PR after the relevant PRs to have the default service.name set. I'm already working on it. :)